components/story-archive-page/StoryArchivePage.style.js

```diff
@@ -63,7 +63,7 @@ const SortingPaginationWrapper = styled.div`
   display: flex;
```

Comment:
Nit: I noticed the archive images don't click on tablet/mobile (only desktop), only the arrow does

Reply:
fixed